### PR TITLE
WIP DO NOT MERGE - Hotfix/5.0.5

### PR DESCRIPTION
**Pull request by davidrajones**
CI results here:
https://casm.pages.internal.sanger.ac.uk/cit-github/pcap-core/results/hotfix-5.0.5/summary/

### CODE REVIEW

File: bin/merge_or_mark.pl [hunk 1]
```diff
@@ -119,6 +120,9 @@ sub setup {
   if($opts{'qnamesort'} && !$opts{'nomarkdup'}){
       die "ERROR: -qnamesort can only be used in conjunction with -nomarkdups\n";
   }
+  if($opts{'noindex'} && !$opts{'qnamesort'}){
+      die "ERROR: -noindex can only be used in conjunction with -qnamesort\n";
```
Comment (keiranmraine) [inline bin/merge_or_mark.pl:124]: This isn't documented in the command line help

**Comment by davidrajones** [comment]
Did I miss somewhere? It's already done here:

https://github.com/cancerit/PCAP-core/blob/24c82159373b943f4c8b1f8b8c88d7713f838734/Dockerfile#L61

**Comment by keiranmraine** [comment]
> Did I miss somewhere? It's already done here:
> 
> https://github.com/cancerit/PCAP-core/blob/24c82159373b943f4c8b1f8b8c88d7713f838734/Dockerfile#L61

I must be going blind

**Comment by davidrajones** [comment]
> > Did I miss somewhere? It's already done here:
> > https://github.com/cancerit/PCAP-core/blob/24c82159373b943f4c8b1f8b8c88d7713f838734/Dockerfile#L61
> 
> I must be going blind

Glad it's not me going mad! Thanks for the quick review.

**Pull request closed without merging**